jenkins-x.yml

```diff
@@ -2,97 +2,89 @@ buildPack: none
 pipelineConfig:
   pipelines:
     overrides:
-    - name: changelog
-      pipeline: release
-      stage: promote
-      step:
-        command: echo "skipping promote"
+      - name: changelog
```

Comment:
Not sure the indent is necessary, was this auto-formatted?

Reply:
It was auto-formatted. Let me disable that and re-add the changes.